corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -51,3 +51,12 @@ def _clear_cache(doc):
         user.clear_quickcache_for_user()
     iter_bulk_delete(CommCareUser.get_db(), get_all_user_ids(), doc_callback=_clear_cache)
     User.objects.all().delete()
+
+
+def get_deleted_by_username(cls, username):
+    result = cls.get_db().view('deleted_users_by_username/view',
+                               key=username,
+                               include_docs=True,
+                               reduce=False
```

Comment:
Assuming using `first` here is better than using `one` since `one` would raise exception in case of no result found, which is expected in case of checking if user was deleted?